src/test/kotlin/no/nav/syfo/cronjob/VerifyPartnerIdCronjobSpek.kt

```diff
@@ -0,0 +1,190 @@
+                        cronJob.verifyPartnerIdJob()
+                    }
+                    val kontor1 = database.getBehandlerKontorById(kontorId1)
+                    kontor1.dialogmeldingEnabled shouldBeEqualTo null
+                    val kontor2 = database.getBehandlerKontorById(kontorId2)
+                    kontor2.dialogmeldingEnabled shouldNotBe null
+                }
+                it("Cronjob gjør ingenting med duplikat kontor med udatert partnerID hvis allerede disabled") {
+                    var kontorId1: Int
+                    var kontorId2: Int
+                    database.connection.use { connection ->
+                        kontorId1 = connection.createBehandlerKontor(
+                            kontor = BehandlerKontor(
+                                partnerId = PARTNERID,
+                                herId = OTHER_HERID,
+                                navn = KONTOR_NAVN,
+                                adresse = null,
+                                postnummer = null,
+                                poststed = null,
+                                orgnummer = null,
+                                dialogmeldingEnabled = false,
+                                dialogmeldingEnabledLocked = false,
+                                system = null,
+                                mottatt = OffsetDateTime.now()
+                            )
+                        )
+                        kontorId2 = connection.createBehandlerKontor(
+                            kontor = BehandlerKontor(
+                                partnerId = OTHER_PARTNERID,
+                                herId = OTHER_HERID,
+                                navn = KONTOR_NAVN,
+                                adresse = null,
+                                postnummer = null,
+                                poststed = null,
+                                orgnummer = null,
+                                dialogmeldingEnabled = true,
+                                dialogmeldingEnabledLocked = false,
+                                system = null,
+                                mottatt = OffsetDateTime.now()
+                            )
+                        )
+                        connection.commit()
+                    }
+                    runBlocking {
+                        cronJob.verifyPartnerIdJob()
+                    }
+                    val kontor1 = database.getBehandlerKontorById(kontorId1)
+                    kontor1.dialogmeldingEnabled shouldBeEqualTo null
+                    val kontor2 = database.getBehandlerKontorById(kontorId2)
+                    kontor2.dialogmeldingEnabled shouldNotBe null
+                }
+                it("Cronjob gjør ingenting med kontor med udatert partnerID hvis det ikke finnes et annet kontor med samme herId") {
+                    val kontorId = database.connection.use { connection ->
+                        connection.createBehandlerKontor(
```

Comment:
Blir en del repetisjon for å lage testdata her som man evt kunne flyttet ut i en metode på `TestDatabase`.

Reply:
Fikset, ble en del bedre ja.